tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -570,6 +570,25 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
     }
   }
 
+  "Encrypted search" should {
+    "tokenize a query correctly" in {
```

Comment:
Might be nice to have a call to tokenizeData and verify that the result of tokenizeQuery is a subset of the result.

Reply:
Yep, I had meant to do that and forgot. On it now

Reply:
Done